geode-core/src/upgradeTest/java/org/apache/geode/internal/net/SocketCreatorUpgradeTest.java

```diff
@@ -244,26 +264,33 @@ public void upgradingToNewGeodeOnOldJavaWithProtocolsTLSv1_2() throws IOExceptio
   }
 
   @Test
-  public void upgradingToNewGeodeOnOldJavaWithProtocolsTLSv1_2Hangs() throws IOException {
-    assumeThat(version).as("Geode 1.12.0 and older can upgrade.")
+  public void upgradingToNewGeodeOnOldJavaWithProtocolsTLSv1_2Hangs()
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    assumeThat(testVersion).as("Geode 1.12.0 and older can upgrade.")
         .isGreaterThan(TestVersion.valueOf("1.12.0"));
-    assumeThat(version).as("Geode between [1.12.1, 1.3.0) can't connect p2p with just TLSv1.2")
+    assumeThat(testVersion).as("Geode between [1.12.1, 1.3.0) can't connect p2p with just TLSv1.2")
         .isGreaterThanOrEqualTo(TestVersion.valueOf("1.13.0"));
 
     generateSecurityProperties(PROTOCOL_TLSv1_2, securityPropertiesFile, keyStoreFile,
         trustStoreFile);
 
     gfshOldGeodeOldJava.execute(root, startLocator1);
-    runAsync(() -> gfshNewGeodeOldJava.execute(root, startLocator2));
+    runAsync(() -> {
+      try {
+        gfshNewGeodeOldJava.execute(root, startLocator2);
+      } catch (IOException | ExecutionException | InterruptedException | TimeoutException ignore) {
+        // ignored
+      }
+    });
```

Comment:
This is verbose and ugly. Consider wrapping checked exceptions inside an unchecked exception.